### DISCUSSION TITLE
Define a search index in the Resource configuration

### DISCUSSION THREAD

**Discussion by bastihilger** [statamic-rad-pack/runway#501] [Feature Requests]
It would be great if we could define a search index, maybe like so in `config/runway.php`:

```php
'resources' => [
    \App\Models\Product::class => [
        'search_index' => 'products',
   ],
],
```

That we then could use like so in `config/statamic/search.php`:

```php
'indexes' => [
    //...

    'products' => [
        'driver' => 'algolia',
        'searchables' => ['runway:product'],
        'fields' => ['title'],
    ],
],
```

**Comment by duncanmcclean** [comment]
Added in #524, which has been released in [v7.5.0](https://github.com/statamic-rad-pack/runway/releases/tag/v7.5.0).
